Formula/imagemagick.rb

```diff
@@ -69,6 +71,9 @@ def install
       --enable-static
     ]
 
+    args << (build.without? "fontconfig") ? "--without-fontconfig" : "--with-fontconfig"
+    args << (build.without? "freetype") ? "--without-freetype" : "--with-freetype"
+
     if build.without? "modules"
```

Comment:
I don't get it. This way it will only work if the user installs with `brew install imagemagick --with-fontconfig`?